invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -261,38 +260,51 @@ export class CreatibutorsModal extends Component {
     return (
       <span key={identifier.identifier}>
         <a href={link} target="_blank" rel="noopener noreferrer">
-          <Image
-            src={icon}
-            className="inline-id-icon ml-5 mr-5"
-            verticalAlign="middle"
-          />
+          <Image src={icon} className="inline-id-icon mr-5" verticalAlign="middle" />
           {identifier.scheme === "orcid" ? identifier.identifier : null}
         </a>
       </span>
     );
   };
 
+  makeSubheader = (creatibutor) => {
+    const { isOrganization } = this.state;
+
+    if (isOrganization) {
```

Comment:
there is a chance to add this in react-invenio-forms:

https://github.com/inveniosoftware/react-invenio-app-ils/blob/master/src/lib/components/SeparatedList/SeparatedList.js and use it for displaying the comma separated lists without replicating the code below